ic_prob: 0.011
jc_prob: 1.0e-5
optimization_rate: 25
